pisa/stages/data/simple_data_loader.py

```diff
@@ -78,7 +78,7 @@ def __init__(self,
         # instead of adding params here, consider making them instantiation
         # args so nothing external will inadvertently try to change
         # their values
-        expected_params = ()
+        expected_params = ('livetime')
```

Comment:
"livetime" shouldn't be listed in this stage as an expected param

Reply:
If I don't list it there, the stage throws an error saying there are too many params. The only way I found to fix this was to define the livetime param in all pipelines (setting it to -1 in simulation), and make it a requirement of the simple_data_loader